recipes/iolabs/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "iolabs" %}
+{% set version = "3.2" %}
+{% set sha256 = "74c8dbc4cd4bfd5e01f8a2655455330cac81dd3de99a766630ecd0896b8be84b" %}
+
+package:
+  name: iolabs
+  version: {{version}}
+
+source:
+  fn: ioLabs-3.2.tar.gz
+  url: https://github.com/ioLab/python-ioLabs/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: true  # [py3k or linux]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - hid
+
+about:
+  home: https://github.com/ioLab/python-ioLabs
+  license: BSD-3
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Sure, as I think you saw it's in the main project tree, but just not included in the manifest. 

This project hasn't been touched since 2010 so I don't know if there will be another release, and don't know if they want to bump the version just for a manifest change. 

The purpose of all of these new conda-forge recipes is to add a complete @psychopy recipe (since that doesn't easily pip install), but I may just close this PR and leave it out of the conda dependency since I believe it pip installs without a problem, and may be cleaner for conda-forge to only have active projects?